website_sale_cache/README.rst

```diff
@@ -7,5 +7,9 @@ That allows to significantly accelerate the loading of the page with a large num
 
 Note
 ----
-When you use multiple odoo workers, the cache is updated separately for each one. So the page may be load slowly until the cache is updated for each worker. After the cache is updated for each worker, the page will load faster, as far as the advantages of caching allow.
+When used with multiple odoo workers, the cache is updated separately for each one. So the page may be load slowly until the cache is updated for each worker. After the cache is updated for each worker, the page will load faster, as far as the advantages of caching allow.
+
+Warning
+----
+Cache updates every time on creation, editing or deletion of instance product public category model. That can cause problems as example when importing a large number of categories. It is recommended to temporarily disconnect the module in such cases.
```

Comment:
Cache is updated every time on creation, editing or deletion the product public category model instance. This can lead to issues, e. g. when importing a large number of categories. It is recommended to temporarily uninstall the module in this case.